l1-contracts/contracts/state-transition/chain-deps/facets/Executor.sol

```diff
@@ -212,7 +215,7 @@ contract ExecutorFacet is ZkSyncHyperchainBase, IExecutor {
 
     /// @inheritdoc IExecutor
     function commitBatches(
-        StoredBatchInfo memory _lastCommittedBatchData,
+        StoredBatchInfo calldata _lastCommittedBatchData,
```

Comment:
@koloz193 Can you please check what is the gas differences for your changes?

Reply:
i updated the comment with the report